lib/tasks/canary.rake

```diff
@@ -0,0 +1,69 @@
+namespace :canary do
+  desc 'Create a new Canary factory auth token'
+  task :create_factory_auth, [:memo, :flock_id] => :environment do |_task, args|
+    abort 'Memo required' if args[:memo].nil?
+
+    require_relative 'rake_helpers/canary_configuration'
```

Comment:
Just wondering if it is worth moving the file here to an initializer!? 🤔  It would be yet another thing to load, slowing startup down further (for tests and rake tasks alike), but would mean we don't need this line (and others below), nor anywhere else we might need to call on the Canary classes generally.

Can always move it later i guess. what do you think?

Reply:
This did occur to me but the configuration isn't required for the app so I didn't think it was suitable.

If `ThinkstCanary` becomes its own gem (a possibility if other teams want to use it) then it would make sense for it to have the `require: false` options in `Gemfile`, and then be required explicitly. An initializer in this case would cause problems.

We can talk about this.